<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>20</y>
      <width>401</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>选择路径</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_12" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>291</width>
      <height>522</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>参数设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>信息启发因子</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="specialValueText">
              <string>2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>期望启发因子</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="specialValueText">
              <string>4</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>全局信息素挥发参数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>31</width>
               <height>31</height>
              </size>
             </property>
             <property name="specialValueText">
              <string>0.6</string>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>局部信息素挥发参数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="specialValueText">
              <string>0.1</string>
             </property>
             <property name="singleStep">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="text">
              <string>状态转移公式中的q0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="specialValueText">
              <string>0.01</string>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>材料规格</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget_9" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>水平宽度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_10" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>垂直高度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_11" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>最大承重</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="minimumSize">
              <size>
               <width>61</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>2000</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
